docs/src/proposals/stake-v2.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Stake Account Version 2
+---
+
+The current stake program is battle-tested, but time has revealed a few important
+issues:
+
+1. Redelegations are not possible
+
+Whales can't easily redelegate their positions, since they do not want to lose
+out on an epoch of rewards.
+
+2. Rewards calculations at epoch boundaries take too long
+
+Since all of the stake accounts simultaneously receive rewards at the start of an epoch,
```

Comment:
To help with motivation, it would be useful to more fully describe "a big load". 
Without more detail here it's difficult to assess the solution proposed further in the proposal to resolve said big load.

Reply:
Certainly, I'll get some numbers for this